_mobile/android.md

```diff
@@ -273,6 +276,34 @@ packagingOptions {
 }
 ```
 
+Also we have to add all transitive dependencies of our aars. As `pytorch_android` depends on `com.android.support:appcompat-v7:28.0.0` and `com.facebook.soloader:nativeloader:0.8.0`, we need to add them. (In case of using maven dependencies they are added automatically from `pom.xml`).
+
+
+## Custom Build
+
+To reduce the size of binaries you can do custom build of pytorch android with only set of operators required by your model.
+This includes two steps: preparing the list of operators from your model, rebuilding pytorch android with specified list.
+
+1\. Preparation of the list of operators.
```

Comment:
Remove period at the end to stay consistent with the numbering in the rest of document